care/emr/api/viewsets/report/report_upload.py

```
@@ -148,6 +153,52 @@ def generate(self, request, *args, **kwargs):
             status=status.HTTP_201_CREATED,
         )
 
+    @extend_schema(
+        request=GenerateReportRequest, responses={200: "Success"}, tags=["report"]
+    )
+    @action(detail=False, methods=["POST"])
+    def preview(self, request, *args, **kwargs):
```

Ruff (0.15.1)

[warning] Unused method argument: `args`

(ARG002)